tests/Tests/600__ai_apps/600__anaconda_commercial_edition/600__anaconda_commercial_edition.robot

```diff
@@ -75,4 +99,60 @@ Get Connect Button Status
   ${button_status}=  Get Element Attribute    xpath://*/footer/*[.='Connect']    aria-disabled
   [Return]   ${button_status}
 
+Get Anaconda Validator Logs
+  Go To  ${OCP_CONSOLE_URL}
+  Login To Openshift    ${OCP_ADMIN_USER.USERNAME}    ${OCP_ADMIN_USER.PASSWORD}    ${OCP_ADMIN_USER.AUTH_TYPE}
+  Maybe Skip Tour
+  Navigate To Page    Workloads    Pods
+  Search Last Item Instance By Title in OpenShift Table  search_term=anaconda-ce-periodic-validator-job-custom-run  namespace=redhat-ods-applications
+  Click Link    xpath://tr[@data-id='0-0']/td[@id='name']/*/a
+  Click Link    Logs
+  Sleep  2
+  Capture Page Screenshot  logs_page.png
+  ${logs_text}=  Get Text    xpath://div[@class='log-window__lines']
+  ${log_rows}=  Text To List  ${logs_text}
+  [Return]  ${log_rows[0]}
+
+Check Anaconda CE Image Build Status
+  [Arguments]  ${target_status}
+  Navigate To Page    Builds    Builds
+  Search Last Item Instance By Title in OpenShift Table  search_term=minimal-notebook-anaconda  namespace=redhat-ods-applications
+  ${ace_build_status}=  Get Text    xpath://tr[@data-id='0-0']/td/*/span[@data-test='status-text']
+  Run Keyword If    "${ace_build_status}" == "Failed"
+  ...    Fail  the Anaconda image build has failed
+  ...  ELSE
+  ...    Should Be Equal    ${ace_build_status}    ${target_status}
+
+Spawn and Check JupyterLab Using Anaconda CE Image
```

Comment:
I think this should be a TC rather than a keyword, as the steps are the actual test steps and in other cases we usually put these in a TC

Reply:
Agree. I'm going to extract a generic keyword from here and put it in the JupyterHubSpawner.robot (i.e., Spawn Notebook From Dashboard With Arguments). The remaining lines of code will be moved in the TC.